src/docs/Components/Pagination/previews/pagination-default.preview.html

```diff
@@ -0,0 +1,27 @@
+<div class="mc-pagination example">
```

Comment:
Same comment as in the **pagination.preview.html** file and can you improve the indentation of this file.